src/azure-cli/setup.py

```diff
@@ -136,6 +136,7 @@
     'javaproperties==0.5.1',
     'jsmin~=2.2.2',
     'jsondiff==1.2.0',
+    'packaging==20.9',
```

Comment:
Use flexible compatible version range in `setup.py` if no impact of bugs in other versions as this file is used by `pip install`. The packages may be shared with other Python software.
```suggestion
    'packaging~=20.9',
```